File: JS/Dota2Senate.test.js
/**
 * @version JAVASCRIPT ECMAScript 6 
 * @author MARIO SAM <eu@mariosam.com.br>
 * @see I would love to work with you instead solving web code tests: hire me!
 * $ node NomeDaClasse.test
 */
import TEST from 'tape'
import { predictPartyVictory } from './Dota2Senate.js'

TEST('Starting Dota2Senate test...', (t) => {
    //Test 1
    let want = "Radiant"
    let got = predictPartyVictory( "RD" )
    t.assert( want === got, "Expect: "+want)
    //Test 2
    want = "Dire"
    got = predictPartyVictory( "RDD" )
    t.assert( want === got, "Expect: "+want)

    t.end()
})

File: JAVA/Dota2SenateTest.java
/** 
 * @version JAVA
 * @author MARIO SAM <eu@mariosam.com.br>
 * @see I would love to work with you instead solving web code tests: hire me!
 * $ mvn clean test -Dtest=your.package.TestClassName
 */ 
package JAVA;

import static org.junit.Assert.assertEquals;
import org.junit.Test;

public class Dota2SenateTest {

    @Test
	public void testDota2Senate() throws Exception {
        //Test 1
        String want = "Radiant";
        String got = Dota2Senate.predictPartyVictory( "RD" );
        assertEquals(want, got);
        //Test 2
        want = "Dire";
        got = Dota2Senate.predictPartyVictory( "RDD" );
        assertEquals(want, got);
	}

}
